src/pages/home/report/ContextMenu/PopoverReportActionContextMenu.tsx

```diff
@@ -87,8 +95,8 @@ function PopoverReportActionContextMenu(_props, ref) {
             }
 
             popoverAnchorPosition.current = {
-                horizontal: cursorRelativePosition.horizontal + x,
-                vertical: cursorRelativePosition.vertical + y,
```

Comment:
How did it work?

Reply:
This logic is only for smaller screens, dimensions are rarely changed on mobile + x,y are almost always 0

Reply:
I meant just the fact that we used `.horizontal` without dereferencing the actual value with `.current`. It was just a buggy code, right?